src/Data/Functor/Linear.hs

```diff
@@ -70,3 +71,6 @@ instance Functor [] where
 instance Traversable [] where
   traverse _f [] = pure []
   traverse f (a : as) = (:) <$> f a <*> traverse f as
+
+instance Functor (Const x) where
+  fmap _ (Const x) = Const x
```

Comment:
It's also `Applicative` (when `x` is a monoid) and `Traversable`. We might as well define these.

Also, we should definitly reexport `Const` (as well as, eventually, many functors) in `Functor.Linear`.

Reply:
If it's okay, I'll hold off on defining the `Applicative` instance in this PR and add it when I add linear monoids.

Reply:
Sure.